documentation/modules/auxiliary/scanner/mssql/mssql_idf.md

```diff
@@ -0,0 +1,59 @@
+## Description
+
+This (Interesting Data Finder) module will connect to a remote MSSQL server using a given set of credentials and search for rows and columns with “interesting” names. This information can help you fine-tune further attacks against the database.
+
+## Verification Steps
+
+1. Do: ```use auxiliary/scanner/mssql/mssql_idf```
+2. Do: ```set RHOSTS [IP]```
+3. Do: ```set THREADS [number of threads]```
+4. Do: ```run```
+
+To configure the module, set it to look for field names of ‘username’ and ‘password’, along with a known password for the system, and our RHOST value.
+
+## Scenarios
+
+**Running the scanner**
```

Comment:
replace w/ 3rd level title of what version of MSSql you scanned and host os if possible